cypress/integration/instant/NewTrade.spec.ts

```diff
@@ -154,9 +154,11 @@ describe('New trade', () => {
     trade.sell('ETH')
       .amount('5');
 
-    makeScreenshots('price-impact-highlight');
-
     trade.expectPriceImpact(`19.28%`, true);
+
+    cy.get(tid('notification-cross')).click();
```

Comment:
is this the only place that it could happen before making screenshots?

Reply:
Yup, I've checked the places where we used makeScreenshots and there rest of the cases didn't have that kind of undeterministic dependency